plugins/obs-ffmpeg/obs-ffmpeg-source.c

```diff
@@ -57,6 +59,9 @@ struct ffmpeg_source {
 	bool is_clear_on_media_end;
 	bool restart_on_activate;
 	bool close_when_inactive;
+
+	bool pause_and_resume; 	// option that says you want the sourced paused on deactivate and resumed on activate (rather than stopped and restarted from begining)
+
```

Comment:
I like the motivation behind the comment, but I don’t think it’s needed since it’s explained further down.
If you think it’s needed, it would be good to make it a little shorter.

Also the newlines aren’t needed

Reply:
ok happy to remove that.